### DISCUSSION TITLE
Some suggestions for sorting songs

### DISCUSSION THREAD

**Discussion by stefan-franz** [strawberrymusicplayer/strawberry#1310] [Ideas]
I' m searching under Linux Mint 21.2 for a program who is nearly Mediamonkey. 
I'm a DJ and for sorting my (new) songs, maybe some improvements will built in the future:

A view with a file directory tree like in a file browser, where i can doubleklick to a mp3 to play and after that to delete or move in the file directory tree. I have about 24.000 Songs on my Laptop and i search for a program who has a sorting feature - at the moment i don't know one unter Linux. 

Would be great if you make such an improvement. 

Party-DJ Stefan

**Comment by Visikde** [comment]
You can do this with dolphin file browser, it will right click open with strawberry  
1st step copy a playlist to any external storage, edit using dolphin, your library will remain complete & your setlist will be on the external media

**Comment by jonaski** [comment]
https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
